src/components/cards/categoryCard/CategoryCard.module.css

```diff
@@ -0,0 +1,33 @@
+.home_product_category {
+  background-color: var(--white);
+}
+
+.home_product_category:hover {
+  transform: scale(1.09);
+  transition: all 0.3s ease-in-out;
+}
+
+.home_product_category_overlay {
+  opacity: 0;
+  font-size: var(--font-s-lg);
+  font-weight: var(--font-w-semibold);
+  padding: 1rem;
+  cursor: pointer;
+  transition: all 0.3s ease-in-out;
+}
+
+.home_product_category:hover .home_product_category_overlay {
+  display: flex;
+  justify-content: center;
+  align-items: center;

```

Comment:
use utility class here

Reply:
Thanks for pointing this out.

Reply:
Actually, I want these properties only on hover that's why not used the utility class.